fhir-persistence/src/test/java/com/ibm/fhir/persistence/search/test/AbstractSearchQuantityTest.java

```diff
@@ -139,42 +144,40 @@ public void testSearchQuantity_Quantity_withPrefix_LE() throws Exception {
 
     @Test
     public void testSearchQuantity_Quantity_withPrefix_GE() throws Exception {
+        // Implicit range of indexed 'Quantity' value is 24.5 to 25.5
         assertSearchReturnsSavedResource("Quantity", "ge24|http://unitsofmeasure.org|s");
         assertSearchReturnsSavedResource("Quantity", "ge24.4999||s");
         assertSearchReturnsSavedResource("Quantity", "ge24.5||s");
         assertSearchReturnsSavedResource("Quantity", "ge25||s");
         assertSearchReturnsSavedResource("Quantity", "ge25.4999||s");
-        // We need to track whether the high is inclusive or not to get this right
-        // For decimals, the high is exclusive, but for Range the high is inclusive
-//        assertSearchDoesntReturnSavedResource("Quantity", "ge25.5||s");
+        assertSearchDoesntReturnSavedResource("Quantity", "ge25.5||s");
```

Comment:
Would be good to have both edges of where the resource is returned vs not returned. ("le 25.0001||s"?)

Reply:
In this case, the edges are search values of 25.5 (won't return resource) and a value less than 25.5 (will return a resource). Already have a 25.5 case and a 25.4999 case, so I think the edges are covered.